frontend/src/app/admin/admin-routing/admin-routing.module.ts

```diff
@@ -6,8 +6,11 @@ import {AdminRolesComponent} from '../components/admin-content/admin-roles/admin
 import {AdminRolesFormComponent} from '../components/admin-content/admin-roles-form/admin-roles-form.component';
 import {AdminLogoutComponent} from '../components/admin-content/admin-logout/admin-logout.component';
 import {AdminProfileComponent} from '../components/admin-content/admin-profile/admin-profile.component';
-import {AdminTypeRequetesComponent} from "../components/admin-content/admin-type-requetes/admin-type-requetes.component";
-import {AdminTypeRequetesFormComponent} from "../components/admin-content/admin-type-requetes-form/admin-type-requetes-form.component";
+import {AdminTypeRequetesComponent} from '../components/admin-content/admin-type-requetes/admin-type-requetes.component';
+import {AdminTypeRequetesFormComponent} from '../components/admin-content/admin-type-requetes-form/admin-type-requetes-form.component';
+import {AdminMotifComponent} from '../components/admin-content/admin-motif/admin-motif.component';
```

Comment:
Please rename these components into `AdminMotifSuppressionComponent` and `AdminMotifSuppressionFormComponent` in order to get it consistent